minio/api.py

```diff
@@ -166,7 +166,7 @@ def _handle_redirect_response(
             307: ("Redirect", "Temporary redirect"),
             400: ("BadRequest", "Bad request"),
         }.get(response.status, (None, None))
-        region = response.getheader("x-amz-bucket-region")
+        region = response.headers.get("x-amz-bucket-region")
```

Comment:
This PR https://github.com/minio/minio-py/pull/1249 fixed and passed CI on `Ubuntu` and `Windows`. You would need to add `macos` in Github workflow to get it passed.

Reply:
Ok, will add the github action with macos-latest like the others although it's not clear the architecture (https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md, I assume still Intel if not specified there...)